packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -714,4 +714,4 @@ div.graphie {
     padding-top: 0;
 }
 
-@import "../../../math-input/less/main.less";
+@import (reference) "@khanacademy/math-input/styles.css";
```

Comment:
This becomes a `(reference)` now to avoid the situation where an app using `math-input` directly would "double" the math-input CSS because it's integrated into the Perseus CSS also. 

This means that consuming apps will need to ensure that math-input CSS is properly imported, but that's a one-line change and allows for deduplication of math-input CSS.